### ISSUE TITLE
Add support for documentation of enum variants in enum generation macros

### ISSUE THREAD

**Issue by Eonm** [jbaublitz/neli#37]
Hi,

It is possible to document each enum variant created with the macro impl_var_trait! ?

```rust
    impl_var_trait!(
    /// nl80211Commands
    ///
    /// Enumeration from nl80211/nl80211.h:880
    Nl80211Cmd, u8, Cmd,
    CmdUnspec                  => 0,
    /// Request information about a wiphy or dump request to get a list of all present wiphys.
    /// How can I document this enum variant ?
    CmdGetWiphy                => 1,
    CmdSetWiphy                => 2
    );
```
Thanks

**Comment by jbaublitz** [comment]
Hi @Eonm, I'm going to be reworking the macros as part of #19. I can certainly add support for variant documentation. I don't tend to use it because of the number of enums I support out of the box and I'm not always entirely sure what each one represents due to lack of netlink documentation. However because it's exposed as part of the library, this makes sense to support. I'll edit this issue and will add this feature as I do #19.

**Comment by Eonm** [comment]
Indeed, lack of documentation seems to be the rule with netlink. 

I tried to figure out how to implement this feature myself, but i never used rust macros before.

Thank you for your reply.

**Comment by jbaublitz** [comment]
This should be resolved in commit b00b0bbc7a7975a0cd43d831a40541c63e0e52df. Closing.
